### PR TITLE
Added missed wait time for pt, ptg operations, added fix for stack no…

### PR DESCRIPTION
**Pull request by ashutosh-mishra**
…t cleaned up issue.

### CODE REVIEW

**Comment by ashutosh-mishra** [comment]
@njagadish @rajendramachani @YogeshRajmane

File: gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/nfp_node_driver.py [hunk 1]
```diff
@@ -192,7 +192,7 @@ def get_network_function(self, context, network_function_id):
     def consumer_ptg_added_notification(self, context, network_function_id,
                                         policy_target_group):
         cctxt = self.client.prepare(version=self.RPC_API_VERSION)
-        cctxt.cast(context,
+        return cctxt.call(context,
```
Comment (njagadish) [inline gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/nfp_node_driver.py:195]: Do we need to check for call timeout? What happens if this call timesout?

Comment (ashutosh-mishra) [inline gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/nfp_node_driver.py:195]: This rpc call will return immideately, after that we are waiting.

**Comment by njagadish** [comment]
Decided to add this change on 5th May branch as 4th May is not in sync with base (29th April got merged after 4th May branch is taken out of base)

**Comment by ashutosh-mishra** [comment]
Raise PR w.r.t another branch https://github.com/oneconvergence/group-based-policy/pull/280

**Pull request closed without merging**